app/src/main/res/values/strings.xml

```diff
@@ -491,43 +491,43 @@
     <string name="firewall_card_configure_btn">block apps</string>
 
-    <string name="dns_card_connected_status_failure">Disabled</string>
+    <string name="dns_card_connected_status_failure">Off</string>
 
     <string name="dns_card_latency_active">Latency: %1$sms</string>
-    <string name="dns_card_latency_inactive">Enable DNS mode.</string>
+    <string name="dns_card_latency_inactive">Turn on DNS mode.</string>
 
-    <string name="apps_card_ips_count">%1$s IP &amp; Port rules</string>
+    <string name="apps_card_ips_count">%1$s IP and port rules</string>
 
     <string name="hsf_start_btn_state">start</string>
     <string name="hsf_stop_btn_state">stop</string>
 
     <string name="hsf_uptime">(active since %1$s; protos: %2$s)</string>
     <string name="hsf_downtime">(inactive since %1$s)</string>
 
-    <string name="hsf_vpn_dialog_header">Attention!</string>
-    <string name="hsf_vpn_dialog_message">RethinkDNS needs permission to set up a local VPN to encrypt the device\'s DNS requests, implement a network monitor, and enforce firewall rules. \n\nRethinkDNS does not collect any information or spy on you in any way.</string>
+    <string name="hsf_vpn_dialog_header">Action needed</string>
+    <string name="hsf_vpn_dialog_message">Grant RethinkDNS permission to set up a local VPN to encrypt the device\'s DNS requests, implement a network monitor, and enforce firewall rules. \n\nRethinkDNS does not collect any info or spy on you in any way.</string>
     <string name="hsf_vpn_dialog_positive">Proceed</string>
     <string name="hsf_vpn_dialog_negative">Cancel</string>
 
-    <string name="hsf_vpn_prepare_failure">Either VPN permission missing, or another app is in Always-on VPN mode.</string>
+    <string name="hsf_vpn_prepare_failure">Either a VPN permission missing, or another app is in always-on VPN mode.</string>
 
     <string name="hsf_start_dialog_header">Start %1$s</string>
-    <string name="hsf_start_dialog_message">You are one step away from safe and secure Android: Instantly see network activity, block websites, IPs, and badly behaving apps.</string>
+    <string name="hsf_start_dialog_message">One step away from blocking websites, and browsing network activity, IPs, and badly behaving apps.</string>
     <string name="hsf_start_dialog_positive">Start</string>
     <string name="hsf_start_dialog_negative">Dismiss</string>
 
     <string name="hsf_whats_new_chip_text">What\'s new</string>
     <string name="hsf_proxy_chip_text">Proxy active</string>
     <string name="hsf_blocklist_chip_text">%1$s blocklists configured</string>
-    <string name="hsf_blocklist_chip_text_no_blocklist">RethinkDNS in-use</string>
-    <string name="hsf_blocklist_chip_text_no_data">Enable RethinkDNS+</string>
+    <string name="hsf_blocklist_chip_text_no_blocklist">RethinkDNS in use</string>
+    <string name="hsf_blocklist_chip_text_no_data">Turn on RethinkDNS+</string>
```

Comment:
pls see prev comments

Reply:
What is the `+` here?
Sometimes it is "Rethink", sometimes "RethinkDNS", and sometimes "Rethink DNS", and sometimes just "DNS".
I get that DNS is not required in Rethink, but beyond that I am at a loss.